packages/trie/examples/merkle_patricia_trees/README.md

```diff
@@ -496,43 +486,43 @@ You might have noticed that this child node is a "hash", while in the previous e
 Similarly to leaf nodes, extension nodes are two-item arrays: `[ encodedPath, hash ]` . The encodedPath (denoted above as "\_nibbles") stands for the "remaining path". In the case of extension nodes this is the path that we "shortcut". Recall our two keys:
 
 ```jsx
-console.log(Buffer.from('testKey0001'))
-console.log(Buffer.from('testKey000A'))
+console.log(bytesToHex(utf8ToBytes('testKey0001')))
+console.log(bytesToHex(utf8ToBytes('testKey000A')))
 
 // RESULT
-<Buffer 74 65 73 74 4b 65 79 30 30 30 31>
-<Buffer 74 65 73 74 4b 65 79 30 30 30 41>
+0x746573744b657930303031
+0x746573744b657930303041
 ```
 
-The first part of the path ("testKey" = `<74 65 73 74 4b 65 79>`) led us to the branch node. Next, taking the branch at index `3` (for hex value `<3>`) led us to our extension node, which automatically leads us down the path `[ 0, 3, 0, 3, 0 ]`. Using only two nodes (branch + extension), we are therefore able to "shortcut" the whole `<30 30 30>` part of the path! With a standard trie, this would have required 6 successive branch nodes!
+The first part of the path ("testKey" = `0x746573744b6579`) led us to the branch node. Next, taking the branch at index `3` (for hex value `3`) led us to our extension node, which automatically leads us down the path `03030`. Using only two nodes (branch + extension), we are therefore able to "shortcut" the whole `303030>` part of the path! With a standard trie, this would have required 6 successive branch nodes!
```

Comment:
```suggestion
The first part of the path ("testKey" = `0x746573744b6579`) led us to the branch node. Next, taking the branch at index `3` (for hex value `3`) led us to our extension node, which automatically leads us down the path `03030`. Using only two nodes (branch + extension), we are therefore able to "shortcut" the whole `303030` part of the path! With a standard trie, this would have required 6 successive branch nodes!
```